webapp/src/components/Conference/Conference.tsx

```diff
@@ -28,26 +30,43 @@ export const Conference = (): JSX.Element => {
     setPipHidden(!pipHidden)
   }
 
+  const videoTracks = localStream?.getVideoTracks()
+  const videoTrackId = videoTracks != null && videoTracks.length !== 0 ? videoTracks[0].id : ''
+
+  // Only re-render the selfie if the videoTrack id changes
+  const selfie = useMemo((): JSX.Element => {
+    return <Selfview localMediaStream={localStream} isVideoInputMuted={false} shouldShowUserAvatar={false} username={''} data-testid='SelfView'/>
+  }, [videoTrackId])
```

Comment:
should you just use `React.memo` instead and create additional component outside of this component?

Reply:
This is to avoid the flickering in the selfview when the user unmutes his microphone. In that case I call to `getUserMedia` but only update the audio tracks.

Reply:
I tried to implement it, but it didn't work. The problem is that the `localStream` changes and the only thing that is preserved is the `videoTrackId`.